### PR TITLE
doc: Cloud Open Beta updates

### PR DESCRIPTION
**Pull request by msudberg**
### Motivation

This PR documents updated features.

### Description

* Folded "What is Materialize Cloud?" into Cloud index page
* Renamed Cloud quickstart to match general Mz Getting Started guide
* Updated Cloud Getting Started and related pages (connection, create deployments, delete deployments)
* Added new Admin, Troubleshooting, Settings pages
* Updated Cloud account limits and support instructions

### Checklist

- [x] This PR has adequate test coverage.
- [x] This PR adds a release note for any user-facing behavior changes.

### CODE REVIEW

File: doc/user/content/cloud/create-deployments.md [hunk 1]
```diff
@@ -9,11 +9,15 @@ menu:
...
-On the [Deployments](https://cloud.materialize.com/deployments) page, click **Create deployment**.
+This functionality is only available to [administrators](administer-workspace).
 
-Materialize creates the deployment and assigns it a name and hostname. This should only take about a minute and a half.
+1. On the [Deployments](https://cloud.materialize.com/deployments) page, click **+New deployment**.
 
-The free version of Materialize Cloud allows two deployments. If you need more, please [contact us](../support).
+1. Select the desired deployment size and click **Create**.
+
+Materialize creates the deployment, automatically assigning it a name and hostname. The deployment should be ready for connections in about a minute and a half.
```
Comment (sinkim-materialize) [inline doc/user/content/cloud/create-deployments.md:18]: @msudberg 

Users will be able to name their deployments as part of the creation flow in Open Beta (versus the current behavior where deployment names are automatically generated).

cc: @benesch

File: doc/user/content/cloud/create-deployments.md [hunk 2]
```diff
@@ -9,11 +9,15 @@ menu:
...
 {{< cloud-notice >}}
 
-On the [Deployments](https://cloud.materialize.com/deployments) page, click **Create deployment**.
+This functionality is only available to [administrators](administer-workspace).
 
-Materialize creates the deployment and assigns it a name and hostname. This should only take about a minute and a half.
+1. On the [Deployments](https://cloud.materialize.com/deployments) page, click **+New deployment**.
 
-The free version of Materialize Cloud allows two deployments. If you need more, please [contact us](../support).
+1. Select the desired deployment size and click **Create**.
```
Comment (sinkim-materialize) [inline doc/user/content/cloud/create-deployments.md:16]: @msudberg 

Default deployment size on creation flow for Open Beta is XS, and they can upgrade (vertically scale) deployments up to S for free. If users know in advance that for their use case they need bigger than size S deployments, they can/ should reach out to our team to discuss details and place the 'order'.

File: doc/user/content/cloud/create-deployments.md [hunk 3]
```diff
@@ -9,11 +9,15 @@ menu:
...
+
+Materialize creates the deployment, automatically assigning it a name and hostname. The deployment should be ready for connections in about a minute and a half.
+
+The free version of Materialize Cloud allows two deployments. The enterprise version is unlimited.
```
Comment (sinkim-materialize) [inline doc/user/content/cloud/create-deployments.md:20]: @msudberg 

Re: "The enterprise version is unlimited" - this is not set in stone, users can determine how many more deployments they will need and at what size, and sort out payments, and place the 'order' by contacting us during the initial run for Open Beta.

cc: @cbern37 @krishmanoh2

File: doc/user/content/cloud/get-started.md [hunk 4]
```diff
@@ -13,23 +15,31 @@ This guide walks you through getting started with Materialize Cloud, from settin
...
 
 * Exploring common patterns like joins and time-windowing
 
-## Sign up, deploy, connect
+## Sign up and deploy
 
 1. Sign up for Materialize Cloud at [https://cloud.materialize.com](https://cloud.materialize.com#signup).
 
-1. Once you [log in](https://cloud.materialize.com), you can use the [Deployments](https://cloud.materialize.com/deployments) page to create, upgrade, or destroy deployments, as well as to obtain the TLS certificates you need to install on your local machine to connect.
+    You can create a Materialize Cloud account or sign in with an existing Google or Github acccount. SSO and 2FA are also available for [enterprise accounts](account-limits).
```
Comment (sinkim-materialize) [inline doc/user/content/cloud/get-started.md:30]: @msudberg 

re: SSO - per @benesch comment https://github.com/MaterializeInc/cloud/issues/700#issuecomment-912213494, SSO is expected to now be out of scope for the go-live for Open Beta. Please will you ensure that documentation reflects this change accordingly.

Comment (benesch) [reply doc/user/content/cloud/get-started.md:30]: So, AIUI, I think the signup form at cloud.materialize.com is going to stay disabled until self-service beta? From what I understood from @craig-materialize, there's a signup flow that marketing has built out that ultimately drops your email into a sales queue, and then someone on the sales team needs to actually unblock the request with an invitation to Materialize Cloud.

Comment (sinkim-materialize) [reply doc/user/content/cloud/get-started.md:30]: This is news to me - no one's approached me to tell me 'this is what we're going to do for the signups'. I'm open to this but let's please discuss and ensure we're on the same page on this today @craig-materialize @benesch. Will ping you on Slack shortly on this. Thank you.

Comment (sinkim-materialize) [reply doc/user/content/cloud/get-started.md:30]: @msudberg update for you on this: @benesch @TonyRippy and I synced on this in the office today, and we have decided that due to concerns from Engineering about the scalability limits of the system and the bandwidth of the team to handle (urgent) inbound requests from users that we will be implementing an in-between step as described by Nikhil above for the signup flow. That is, users will have to get on a call with us in order to be allowed to proceed to completing the new account creation flow.

cc: @craig-materialize

File: doc/user/content/cloud/account-limits.md [hunk 5]
```diff
@@ -1,31 +1,48 @@
...
+Trials last for 30 days or until you max out the available resources for free deployments, whichever comes first. Accounts and deployments may be deleted 14 days after your trial expires unless you have upgraded to an enterprise account.
+
+## Materialize Cloud specifications
+
+Deployment Size | XS | S | M | L | XL
+----------------|----|---|---|---|---
+**Level**  | free  | free  | enterprise  | enterprise  |  enterprise
+**CPUs**  | 4vCPUs  | 8vCPUs  | 16vCPUs  |  32vCPUs  |  64vCPUs
+**RAM**  |  32GB | 64GB  | 128GB  | 256GB  |  512GB
+**R5B instances**  | r5.xlarge   | r5.2xlarge  | r5.4xlarge   | r5.8xlarge   |  r5.16xlarge
```
Comment (benesch) [inline doc/user/content/cloud/account-limits.md:23]: @rjnn do we want to publicize this?

File: doc/user/content/cloud/administer-workspace.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    weight:
+aliases:
+  - edit-deployments
+---
+
+{{< cloud-notice >}}
+
+Administrative users can set up shared workspaces and add and remove other users.
+
+To change a deployment name or size, click on the dots menu and select **Edit**.
```
Comment (benesch) [inline doc/user/content/cloud/administer-workspace.md:17]: This is just an edit button on the detail page now, not hidden behind a dots menu.

File: doc/user/content/cloud/administer-workspace.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+To add users or remove users, click on the profile icon in the top navbar and select **Settings**.
```
Comment (benesch) [inline doc/user/content/cloud/administer-workspace.md:19]: s/Settings/Account settings/.

File: doc/user/content/cloud/delete-deployments.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+{{< cloud-notice >}}
+
+This functionality is only available to [administrators](../administer-workspace).
+
+Deleting a deployment destroys all data and catalog items. They cannot be restored.
+
+1. Go to the [Deployments](https://cloud.materialize.com/deployments) page.
+2. Click **Delete**.
+3. Enter the confirmation phrase, then click **Yes, destroy my deployment**.
```
Comment (benesch) [inline doc/user/content/cloud/delete-deployments.md:16]: The instructions here don't quite match the UI anymore. The buttons are both called "Destroy", and you have to navigate to a particular deployment detail page; they're no longer accessible from the list.

File: doc/user/content/cloud/get-started-with-cloud.md [hunk 9]
```diff
@@ -13,23 +16,35 @@ This guide walks you through getting started with Materialize Cloud, from settin
...
+1. Sign up for Materialize Cloud at [https://cloud.materialize.com](https://cloud.materialize.com/signup/).
 
-1. Once you [log in](https://cloud.materialize.com), you can use the [Deployments](https://cloud.materialize.com/deployments) page to create, upgrade, or destroy deployments, as well as to obtain the TLS certificates you need to install on your local machine to connect.
+1. Once your account has been created, [log in](https://cloud.materialize.com).
 
-   By default, you can create up to two deployments. If you're interested in more, [let us know](../support)!
+1. If you've been invited to an existing workspace, you can go directly to the [connection instructions](#connect). Otherwise, a dialog asks you to create a workspace. Enter a name and click **Next**.
 
-1. On the [Deployments](https://cloud.materialize.com/deployments) page, click **Create deployment**. Materialize creates a deployment and assigns it a `name` and `hostname`.
+1. Materialize Cloud asks if you want to invite other users. For the purposes of this demonstration, just click **Skip**. (You can always add other users later.)
```
Comment (benesch) [inline doc/user/content/cloud/get-started-with-cloud.md:35]: This bit doesn't presently exist.

File: doc/user/content/cloud/get-started-with-cloud.md [hunk 10]
```diff
@@ -13,23 +16,35 @@ This guide walks you through getting started with Materialize Cloud, from settin
...
+
+1. On the **New deployment** tab, enter the name of your new deployment in the **Title**  field and click **Create**.
```
Comment (benesch) [inline doc/user/content/cloud/get-started-with-cloud.md:37]: s/New deployment tab/Create deployment modal/

File: doc/user/content/cloud/update-deployments.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Update Deployments"
+description: "Update Materialize Cloud."
+aliases:
+  - /cloud/upgrade-deployments
+---
+
+{{< cloud-notice >}}
+
+When a new version of Materialize Cloud is available, an **Update available** notice appears on the [Deployments](https://cloud.materialize.com/deployments) page. Click the notice and enter the requested confirmation to update the deployment.  We recommend updating a single deployment first and testing to ensure that your materializations and computations continue to work as expected.
```
Comment (benesch) [inline doc/user/content/cloud/update-deployments.md:10]: maybe s/updating a single deployment first/updating a non-production deployment first/

**Review by benesch — COMMENTED** [review]
Just dropping in a few notes based on UI changes! Happy to make those changes myself if easier, just lmk!

**Comment by benesch** [comment]
Oh, it also occurs to me: we should get the maintenance window documented, assuming that we've settled on the date/time. Last I heard the proposal was 4:30-5:30pm EST on Wednesdays. During that hour we may reboot your instance repeatedly.

File: doc/user/content/cloud/delete-deployments.md (hunk 8 above)
Comment (morsapaes) [reply doc/user/content/cloud/delete-deployments.md:16]: I'll do a second pass now to catch this kind of thing (not covered in #8216). Thanks, Nikhil!

File: doc/user/content/cloud/account-limits.md (hunk 5 above)
Comment (morsapaes) [reply doc/user/content/cloud/account-limits.md]: The "R5B" bit was removed.

File: doc/user/content/cloud/administer-workspace.md (hunk 7 above)
Comment (morsapaes) [reply doc/user/content/cloud/administer-workspace.md]: One thing I couldn't figure out was how to "add and remove users" (assuming I'm the admin of my own account!).

File: doc/user/content/cloud/get-started-with-cloud.md (hunk 9 above)
Comment (morsapaes) [reply doc/user/content/cloud/get-started-with-cloud.md:35]: Ah, I guess this covers my previous comment.

File: doc/user/content/cloud/administer-workspace.md (hunk 7 above)
Comment (benesch) [reply doc/user/content/cloud/administer-workspace.md]: Oh, yeah, we just haven’t enabled this part yet. It’ll be in account settings when we turn it on.

**Comment by morsapaes** [comment]
Pushed some updates to the content, but structure-wise the docs still feel very sparse. I'd still propose to merge the following standalone pages into existing ones (at least until there's enough content to branch them out):

Administer a Workspace -> Cloud Deployments
Security -> Connect to Cloud

(CC: @naveensukumar)

File: doc/user/content/cloud/_index.md [hunk 12]
```diff
@@ -2,4 +2,30 @@
...
+  - what-is-materialize-cloud
 ---
+
+# What is Materialize Cloud?
+
+{{< cloud-notice >}}
+
+Materialize Cloud hosts and maintains [Materialize](/overview/what-is-materialize) deployments for you, **automating administration tasks** like hardware provisioning, database setup, upgrades, and backups. You can sign up for an account in less than 30 seconds and try out Materialize with your own streaming (and historical) data sources.
+
+{{< cta href="https://cloud.materialize.com/signup" >}}
```
Comment (benesch) [inline doc/user/content/cloud/_index.md:17]: This link'll actually be cloud.materialize.com/account/sign-up.

File: doc/user/content/cloud/security.md [hunk 13]
```diff
@@ -1,23 +1,23 @@
...
 Currently, Materialize Cloud secures your connections by enforcing that clients connect via
-[TLS 1.2+](https://en.wikipedia.org/wiki/Transport_Layer_Security). In the future, the managed
+[TLS 1.2+](https://en.wikipedia.org/wiki/Transport_Layer_Security). In the future, the enterprise
 Materialize Cloud product will offer additional security options, such as:
 
 * Two-factor authentication
 * Encryption at rest
-* Connection over VPC and VPN
+* Secure network ingress and egress
 * Single sign-on integration (SSO)
```
Comment (benesch) [inline doc/user/content/cloud/security.md:21]: We actually offer all of these _expect_ for secure network ingress/egress.

File: doc/user/content/cloud/settings.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+* Name
+* Phone number
+* Address
+* Job title
+
+You can also delete a profile if you wish to delete your Materialize Cloud account. If you are the sole administrator of any workspaces, when you delete your profile, you will be asked whether you want to delete the workspace as well or give administrator privileges to another user.
+
+## Privacy &amp; security
+
+On the Privacy & Security tab, you can change your password. If you are an enterprise user, you can also enable or disable two-step verification.
```
Comment (benesch) [inline doc/user/content/cloud/settings.md:30]: Everyone can do MFA, actually.

File: doc/user/content/cloud/settings.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+
+On the Profile tab, you change any of the following:
+
+* Email address
+* Name
+* Phone number
+* Address
+* Job title
+
+You can also delete a profile if you wish to delete your Materialize Cloud account. If you are the sole administrator of any workspaces, when you delete your profile, you will be asked whether you want to delete the workspace as well or give administrator privileges to another user.
```
Comment (benesch) [inline doc/user/content/cloud/settings.md:26]: Heh, you actually can't do this right now! Maybe: "email us if you need to delete your account"?

**Comment by morsapaes** [comment]
Another suggestion for the future is that we add some screenshots instead of just verbally describing where to find and click things, @sinkim-materialize @msudberg.

**Comment by msudberg** [comment]
@morsapaes - Thanks again for all the improvements!

The "Administer a workspace" page is basically a stub for administration features that were referenced but not detailed in the initial mockups. I could go either way with "Security"; right now it's its own page just to draw attention to it.

I have to admit to a typical tech writer dislike of screenshots, which go out of date very quickly and become a maintenance hazard. But I could add one to the "Cloud deployments" page and then keep the rest of the instructions text-only.

**Pull request merged**